src/vm/arm/exceparm.cpp

```diff
@@ -102,8 +102,10 @@ AdjustContextForVirtualStub(
     PCODE callsite = GetAdjustedCallAddress(GetLR(pContext)); 
 
     // Lr must already have been saved before calling so it should not be necessary to restore Lr
-
-    pExceptionRecord->ExceptionAddress = (PVOID)callsite;
+    if (pExceptionRecord != NULL)
```

Comment:
Do we need the same condition in the x86 version as well?

Reply:
Right, we do.